scripts/nes_trace_parser.py

```diff
@@ -0,0 +1,445 @@
+trace = """
```

Comment:
Interesting, I'm currently working on something similar

Reply:
Do you want me to not have it in the PR?

Reply:
Yes lets remove it for now.